subxt/src/error/dispatch_error.rs

```diff
@@ -191,17 +191,24 @@ pub struct ModuleErrorDetails<'a> {
 /// **Note**: Structure used to obtain the underlying bytes of a ModuleError.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct RawModuleError {
-    /// Index of the pallet that the error came from.
-    pub pallet_index: u8,
-    /// Raw error bytes.
-    pub error: [u8; 4],
+    /// The raw bytes of the module error.
+    ///
+    /// Bytes representation:
+    ///  - `bytes[0]`:   pallet index
+    ///  - `bytes[1]`:   error index
+    ///  - `bytes[2..]`: 3 bytes specific for the module error
+    pub bytes: [u8; 5],
```

Comment:
Let's not expose `bytes` like this, because the `RawModuleError` structure is copied from Substrate (with a `pallet_index` and `error` field as seen before), soooo how about:

- Remove `RawModuleError` entirely now that it doesn't align with the Substate type anymore anyway (and its easiewr for us just to keep all of the bytes for easy decoding etc)
- Add `bytes()`, `pallet_index()` and `error_index()` to `ModuleError` so that one can still obtain the relevant parts and doesn't need to care about any internal details.

Reply:
That makes sense! Have removed the `RawModuleError` thanks!